tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/Serilog/DirectSubmission/LoggerConfigurationInstrumentation.cs

```diff
@@ -66,6 +66,11 @@ private static void TryAddSink<TTarget>(TTarget instance)
             if (!sinkAlreadyAdded)
             {
                 var targetType = instance.Type.Assembly.GetType("Serilog.Core.ILogEventSink");
+                if (targetType is null)
+                {
+                    return;
```

Comment:
nit: I might be useful to log here that we couldn't find the type, as this is something that shouldn't happen

Reply:
Message added in https://github.com/DataDog/dd-trace-dotnet/pull/2716/commits/7a8a8942d10547d8c3d564471e01f9631f2c635d